flink-filesystems/flink-hadoop-fs/src/test/java/org/apache/flink/runtime/fs/hdfs/HadoopFsFactoryTest.java

```diff
@@ -61,4 +72,61 @@ public void testCreateHadoopFsWithMissingAuthority() throws Exception {
 			assertTrue(e.getMessage().contains("authority"));
 		}
 	}
+
+	@Test
+	public void testReadBufferSize() throws Exception {
+		final URI uri = URI.create("hdfs://localhost:12345/");
+		final Path path = new Path(uri.getPath());
+
+		org.apache.hadoop.fs.FileSystem hadoopFs = mock(org.apache.hadoop.fs.FileSystem.class);
+		org.apache.hadoop.fs.FSDataInputStream hadoopInputStream =
+			mock(org.apache.hadoop.fs.FSDataInputStream.class);
+		when(hadoopFs.open(isA(org.apache.hadoop.fs.Path.class), anyInt()))
+			.thenReturn(hadoopInputStream);
+
```

Comment:
We actually want to move away from mocking with mockito as far as possible.
I'm wondering what's the benefit over using `file://` to some random (or temp) file?